### PR TITLE
Documentation edits

### PR DESCRIPTION
**Pull request by duncangeere**
I've reworked the index, install and fundamentals documentation.

- In `index.rst` I've fixed a couple of small typos and improved the clarity of a line.
- In `install.rst` I've re-ordered the OS list so Mac and Windows are at the top, on the grounds that most people will be using those platforms, and that Linux/RPi users are less likely to need help. Also made a small clarity tweak.
- In `fundamentals.rst` I've done a substantive line edit, fixing typos and improving the clarity and communication where possible. Structure is unchanged.

### CODE REVIEW

File: docs/fundamentals.rst [hunk 1]
```diff
@@ -63,26 +52,26 @@ Help on each command is also available on the CLI, for example::
...
 ================
 
-The geometries passed from command to command is organised as a collection of layers, each containing a collection of paths.
+The geometries passed from command to command are organised as a collection of layers, each containing a collection of paths.
 
 .. image:: images/layers.svg
    :width: 300px
 
-The primary purpose of layers is to create or process files for multi-color plots, where each layer contains geometries to be drawn with the specific pen or color. In *vpype*, layers are identified by a non-zero, positive integer (e.g. 1, 2,...). There can be an arbitrary number of layers, memory permitting.
+The primary purpose of layers in *vypype* is to create or process files for multicolored plots, where each layer contains geometries to be drawn with a specific pen or color. In *vpype*, layers are identified by a non-zero, positive integer (e.g. 1, 2,...). You can have as many layers as you want, memory permitting.
```
[the remark this answers is not in the document]
Comment (duncangeere) [reply docs/fundamentals.rst]: Think I've fixed this in my fork - do I need to send another pull request or does it automatically update it?

**Comment by duncangeere** [comment]
Once the typo thing is resolved I'm happy for you to merge! I'll work on the cookbook next, but that's a new thing really.

File: docs/fundamentals.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (abey79) [reply docs/fundamentals.rst]: All commits made on the branch used to create the PR (in this case, your fork's master branch) automatically show up/are included in the pull request, so it's all good.

Side note: usually one creates a dedicated branch for each PR. I'm not 100% if there is any actual issue by using your master, but at least you should make sure it's synced to abey79/vpype@master before doing a new PR.

[the remark this answers is not in the document]
Comment (duncangeere) [reply docs/fundamentals.rst]: Ah okay! I'll do that next time. Thanks, I'm new to all this :)

[the remark this answers is not in the document]
Comment (abey79) [reply docs/fundamentals.rst]: The whole git/github thing does have a bit of a learning curve :) Probably good skills to invest in though...

**Pull request merged**